pynestml/codegeneration/resources_nest/point_neuron/common/NeuronClass.jinja2

```diff
@@ -229,6 +229,14 @@ namespace nest
 //   Node initialization functions
 // ---------------------------------------------------------------------------
 
+#if NEST2_COMPAT
```

Comment:
Would it be better to keep the template files in separate directories based on the version instead of having `#if` directives? Otherwise, we need to add these directives for each NEST released version, correct?

Reply:
I would much prefer separate directories. That would also allow `diff`ing templates much more easily.